models/readme.md

```diff
@@ -1 +1,164 @@
-The MLAgility model corpus will go here.
\ No newline at end of file
+# MLAgility Benchmark
+
+This directory contains the MLAgility benchmark, which is a large collection of models that can be evaluated using the [`benchit` CLI tool](https://github.com/groq/mlagility/blob/main/docs/benchit_user_guide.md).
+
+## Table of Contents
+
+- [Benchmark Organization](#benchmark-organization)
+- [Running the Benchmark](#running-the-benchmark)
+- [Model Template](#model-template)
+
+## Benchmark Organization
+
+The MLAgility benchmark is made up of several corpora of models (_corpora_ is the plural of _corpus_... we had to look it up too). Each corpus is named after the online repository that the models were sourced from. Each corpus gets its own subdirectory in the `models` directory. 
+
+The corpora in the MLAgility benchmark are:
+- `diffusers`: models from the [Huggingface `diffusers` library](https://huggingface.co/docs/diffusers/index), including the models that make up Stable Diffusion.
+- `graph_convolutions`: Graph Neural Network (GNN) models from a variety of publications. See the docstring on each .py file for the source.
+- `popular_on_huggingface`: hundreds of the most-downloaded models from the [Huggingface models repository](https://huggingface.co/models).
+- `selftest`: a small corpus with small models that can be used for testing out the MLAgility tools.
+- `torch_hub`: a variety of models, including many image classification models, from the [Torch Hub repository](https://github.com/pytorch/hub).
+- `torchvision`: image recognition models from the [`torchvision` library](https://pytorch.org/vision/stable/index.html).
+  - _Note_: the `torchvision` library also includes many image classification models, but we excluded them to avoid overlap between our `torch_hub` and `torchvision` corpora.
+- `transformers_keras`: Transformer models from the [Huggingface `transformers` library](https://huggingface.co/docs/transformers/index), expressed in the Keras framework.
+- `transformers_pytorch`: Transformer models from the [Huggingface `transformers` library](https://huggingface.co/docs/transformers/index), expressed in the PyTorch framework.
```

Comment:
There is no one-to-one match between the `transformers_keras` and the `transformers_pytorch` corpora. However, overlaps do exist.

Reply:
Added!